### DISCUSSION TITLE
decrease the number of element

### DISCUSSION THREAD

**Discussion by enghorbani** [neperfepx/neper#692] [General]
Hello 

I already asked how to decrease the number of elements in my spherical particle. I found I can use this command: rcl

I used it, but the number of elements decrease from 27000 to 14000. it is too much and takes lots of time during my simulation in Abaqus.

I need to decrease the number of elements to 1000. Is there any way to decrease the element more?

Thank you

**Comment by rquey** [comment]
For large rcl values, Neper will generally tend to the real minimal number of elements for the geometry of concern. So, it will be difficult to go from 14000 to 1000.

For a more specific answer, please provide your Neper commands / tess file.
